### PR TITLE
Add recipe for psy-view

### PR DESCRIPTION
**Pull request by Chilipp**
This PR adds a recipe for psy-view, a ncview-like interface to open netCDF files, but based on the psyplot framework.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/psy-view/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  description: |
+    This package provides a graphical user interface to quickly visualize the
+    contents of a netCDF file
+  doc_url: http://psyplot.readthedocs.io/projects/psy-view
+  dev_url: https://github.com/psyplot/psy-view
+
+extra:
+  recipe-maintainers:
+    - Chilipp
+
```
Comment (Chilipp) [inline recipes/psy-view/meta.yaml:58]:
```suggestion
```

**Comment by Chilipp** [comment]
@conda-forge/help-python, from my perspective this PR is ready to be merged.

File: recipes/psy-view/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{% set name = "psy-view" %}
+{% set version = "0.0.1b0" %}
```
Comment (beckermr) [inline recipes/psy-view/meta.yaml:2]: We don't generally accept beta versions, but have not had consistent policies either. @conda-forge/core what do we do here again?

Comment (Chilipp) [reply recipes/psy-view/meta.yaml:2]: Thanks for the info @beckermr! The first stable version will come pretty soon as the basis die the software (psyplot, etc.) is already around for quite a while. We just made this a beta version to signalize that the layout of the GUI and implemented features are subject to discussion with the community.

Comment (beckermr) [reply recipes/psy-view/meta.yaml:2]: Understood. We dealt with this case recently and have a policy now. I am simply forgetting it.

Comment (Chilipp) [reply recipes/psy-view/meta.yaml:2]: @beckermr, is this what you are looking for?

https://github.com/conda-forge/cfep/blob/master/cfep-05.md#exceptions

Comment (Chilipp) [reply recipes/psy-view/meta.yaml:2]: seems to me like it is generally allowed to publish pre-release packages on the conda-forge channel, as long there has not yet been a non-beta/alpha release, yet (which applies for psy-view). As soon as there is a non-beta/alpha release for the package, pre-releases need to get special labels

**Pull request merged**